dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/McpTransport.java

```diff
@@ -142,6 +144,17 @@ public McpRequest getMcpRequest() {
         return this.mcpRequest;
     }
 
+    /*
+     * Method for sending initialize response of sessionId as header
+     */
+    public void setResponseHeader(String name, String value) {
```

Comment:
```suggestion
    /*
     * Sets an HTTP header in the response.
     * <p>
     * Intended only for sending the Session ID header with initialize response
     */
    public void setResponseHeader(String name, String value) {
```

Javadoc should first say what the method does and should usually start with a verb phrase. I wouldn't usually include why we added it, but I think it's justified here to encourage us to think before using it for anything else.